src/nvim/eval/typval.c

```diff
@@ -1201,16 +1201,15 @@ void tv_dict_watcher_notify(dict_T *const dict, const char *const key,
   typval_T rettv;
 
   QUEUE *w;
-  QUEUE_FOREACH(w, &dict->watchers) {
+  QUEUE_FOREACH(w, &dict->watchers, {
     DictWatcher *watcher = tv_dict_watcher_node_data(w);
     if (!watcher->busy && tv_dict_watcher_matches(watcher, key)) {
       rettv = TV_INITIAL_VALUE;
       watcher->busy = true;
       callback_call(&watcher->callback, 3, argv, &rettv);
       watcher->busy = false;
-      tv_clear(&rettv);
```

Comment:
Oops! `tv_clear(&rettv)`; seems to be leaking in my commit https://github.com/neovim/neovim/pull/11189/commits/54ce37f5698ce6471adf9b373d1dd146c65b29dc.

Reply:
Added in a4436ba - is that correct?

Reply:
Yes, that ’s fine.